### PR TITLE
ci: bump lint task to bookworm for git v2.38

### PR DESCRIPTION
**Pull request by glozow**
Since 5497c14, verify-commits.py uses `git merge-tree` which requires git v2.38 or later. Fix the lint jobs on master (e.g. https://cirrus-ci.com/task/4971007513985024).

### CODE REVIEW

**Review by hebasto — APPROVED** [review]
ACK 836720e366f61f23c7a56775e11d58890b002d75

File: .cirrus.yml [hunk 1]
```diff
@@ -61,10 +61,10 @@ compute_credits_template: &CREDITS_TEMPLATE
   use_compute_credits: $CIRRUS_REPO_FULL_NAME == 'bitcoin/bitcoin' && $CIRRUS_PR != ""
 
 task:
-  name: 'lint [jammy]'
+  name: 'lint [lunar]'
   << : *BASE_TEMPLATE
   container:
-    image: ubuntu:jammy
+    image: ubuntu:lunar
```
[the remark this answers is not in the document]
Comment (glozow) [reply .cirrus.yml:67]: makes sense to me if bookworm is stable

**Comment by glozow** [comment]
> lgtm ACK https://github.com/bitcoin/bitcoin/commit/257dfef897db0ece387511d0592076381aaeff2b

sorry, pushed again because commit message still said lunar

**Comment by achow101** [comment]
ACK a984beeca10e3ae1ceb3ea53e4dea778160e7079

**Review by hebasto — APPROVED** [review]
re-ACK a984beeca10e3ae1ceb3ea53e4dea778160e7079

**Review by maflcko — APPROVED** [review]

**Pull request merged**